docs/ossm/injection/README.md

```diff
@@ -49,81 +49,183 @@ The injector is configured with the following logic:
+
+1. Once restarted, verify that the excluded pod does not contain a sidecar container, and shows "1/1" containers "Running":
     ```bash
-    oc label namespace default istio-injection=enabled
+    oc get pods -n bookinfo
+    NAME                              READY   STATUS    RESTARTS   AGE
+    details-v1-6bc7b69776-7f6wz       1/1     Running   0          7s
+    productpage-v1-54f48db985-gd5q9   2/2     Running   0          29m
+    ratings-v1-5d645c985f-xsw7p       2/2     Running   0          29m
+    reviews-v1-bd5f54b8c-zns4v        2/2     Running   0          29m
+    reviews-v2-5d7b9dbf97-wbpjr       2/2     Running   0          29m
+    reviews-v3-5fccc48c8c-bjktn       2/2     Running   0          29m
     ```
-1. Injection occurs at pod creation time. Remove the running pod to be injected with a proxy sidecar. 
+
+### Example 3: Enabling sidecar injection with pod labels
+
+Rather than including all workloads within a namespace, you can include individual workloads for sidecar injection. This approach is ideal when only a few workloads within a namespace will be part of a service mesh. 
+
+This example also demonstrates the use of a revision label for sidecar injection. In this case, the `Istio` resource has been created with the name "my-mesh". A unique resource `Istio` name is needed when there are multiple Istio control planes present in the same cluster, or a revision based control plane upgrade is in progress.
+
+Procedure:
+
+1. Verify the revision name of the Istio control plane:
+
+    ```console
+    $ oc get istiorevision
+    NAME      TYPE    READY   STATUS    IN USE   VERSION   AGE
+    my-mesh   Local   True    Healthy   False    v1.23.0   47s
+    ```
+    Since the revision name is `my-mesh`, we must use the a revision label to enable sidecar injection. In this case, `istio.io/rev=my-mesh`.
+
+1. For workloads already running, verify that they show "1/1" containers as "READY", indicating that the pods are currently running without sidecars:
+
     ```bash
-    oc delete pod -l app=sleep
+    $ oc get pods -n bookinfo
+    NAME                             READY   STATUS    RESTARTS   AGE
+    details-v1-65cfcf56f9-gm6v7      1/1     Running   0          4m55s
+    productpage-v1-d5789fdfb-8x6bk   1/1     Running   0          4m53s
+    ratings-v1-7c9bd4b87f-6v7hg      1/1     Running   0          4m55s
+    reviews-v1-6584ddcf65-6wqtw      1/1     Running   0          4m54s
+    reviews-v2-6f85cb9b7c-w9l8s      1/1     Running   0          4m54s
+    reviews-v3-6f5b775685-mg5n6      1/1     Running   0          4m54s
     ```
-1. Verify a new pod is created with the injected sidecar. The original pod has `1/1 READY` containers, and the pod with injected sidecar has `2/2 READY` containers.
+
+1. To find your deployments use the oc get command. For example, to view the Deployment YAML file for the 'ratings-v1' microservice in the bookinfo namespace, use the following command to see the resource in YAML format.
```

Comment:
Steps 3, 4, 5 could be merged into one if using `oc edit deployment -n bookinfo ratings-v1`

Reply:
True, people may have different preferences for editing K8s YAML though, so I think it's ok just to say edit the deployment and then show the example YAML. I think this command may open an unfamiliar editor depending on the user and system they are on (probably vi).

The first step of just doing a "get" on the deployment seemed unnecessary though, so I removed it. I think I copied it from the OSSM 2 docs.